app/assignment/directives/instructorAssignmentTable.js

```diff
@@ -540,6 +562,34 @@ let instructorAssignmentTable = function ($rootScope, AssignmentActionCreators,
 
 			// end on event 'assignmentStateChanged'
 
+			// Handle input box edits
+			element.on("keydown", function(e) {
+				let $el = $(e.target);
+				if ($el.data('event-type') != 'setInstructorNote') { return; }
+
+				if (e.key == 	"Enter") {
+					var instructorId = $el.data('instructor-id');
+					var scheduleId = $el.data('schedule-id');
+
+					var note = e.target.value;
+
+					AssignmentActionCreators.createOrUpdateInstructorNote(scheduleId, instructorId, note);
+				}
+			});
+
+			// Handle input box edits
+			element.on("change", function(e) {
```

Comment:
Does 'change' handle blur? That seems like a logical time to save.

Reply:
As I understand it "change" is similar to "blur", but only fires if the value was actually changed, whereas blur would fire regardless.
https://api.jquery.com/change/

Reply:
Cool, thanks!